src/components/calendar/calendar.js

```diff
@@ -224,6 +224,19 @@ export const BCalendar = Vue.extend({
     labelHelp: {
       type: String,
       default: () => getComponentConfig(NAME, 'labelHelp')
+    },
+    formatLong: {
```

Comment:
This might be better named `formatOptions`

Reply:
Agree. What about `formatText`? Otherwise users may think it's applicable to other displayed text elements as well.

Reply:
maybe `dateFormatOptions` ?

Reply:
But if an example was added to the docs to show usage (under the styling section), that would help users understand.  Or add to the prop description that it formats the date string shown to users.

But docs might be better, showing possible values for each property (weekday, year, month, and day)